### ISSUE TITLE
Getting erraneous output in class predicted by Tensorflow Object detection API with pretrained model.

### ISSUE THREAD

**Issue by nilay0396** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#421]
Hi,
I am using the Tensorflow object detection api with faster_rcnn_inception_v2_coco_2018_01_28 pretrained model. I am using the default 90 classes and frozen_inference_graph.pb and the labelmap.pbtxt file from the obtained pretrained model.
Labelmap.pbtxt has Keyboard with id 76 , TV with id 73 and laptop with id 72.
When i run my object detection code, I expected classes (list with length 90) to output label id of predicted objects.But I have 'Keyboard' , 'TV' and 'Laptop'  detected in my frame and the corresponding output being
Class[[76. 73. 72. 73. 76. 73. 72. 72.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.
   1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.
   1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.
   1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.
   1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.  1.
   1.  1.  1.  1.  1.  1.  1.  1.  1.  1.]]
should'nt it be something like class list with a single entry of 76,72, 73 and the rest of the entries 1.
